### DISCUSSION TITLE
Removing aqueduct without removing the road that passes below?

### DISCUSSION THREAD

**Discussion by mathmaj** [Keriew/augustus#299] [General]
Hi!

Do you think it would be possible to remove a section of aqueduct crossing a road without removing the road?
Like when I remove a barrage?

Thx in advance! :)

**Comment by crudelios** [comment]
Technically it is possible, but what if you want to remove the road without removing the aqueduct? Which one should we prioritise?

Deleting both at once avoids those kinds of issues, and it's also consistent: for the entire game, clearing land does exactly what the name implies. If we start adding exceptions it will become confusing.

**Reply by mathmaj** [reply]
Thanks for your reply! 
I totally agree with your point. I didn't think about it.
